.travis/install.sh

```diff
@@ -7,7 +7,7 @@ if [[ "$(uname -s)" == 'Darwin' ]]; then
     sw_vers
     brew update || brew update
 
-    brew outdated openssl || brew upgrade openssl
+    brew outdated openssl@1.1 || brew upgrade openssl@1.1
```

Comment:
`brew outdated` returns `0` if the argument isn't installed. I think you can just make this `brew install openssl@1.1`, it's not installed in the base image ATM.